app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.design.widget.Snackbar;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button loginBtn;
+    EditText etEmail, etPassword;
+    TextInputLayout tiEmail, tiPassword;
+    Validation validate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_screen);
+
+        loginBtn = (Button) findViewById(R.id.btn_login);
```

Comment:
Works well but you can avoid the `findViewById`s by using the databinding features of the Android framework (https://developer.android.com/topic/libraries/data-binding/index.html) or using a library like Butter Knife (http://jakewharton.github.io/butterknife/). Check them out. :)